### PR TITLE
Update Vietnamese Translation P5

### PR DESCRIPTION
**Pull request by Nad-SK**
Towns name and more

### CODE REVIEW

**Review by ihhub — CHANGES_REQUESTED** [review]
Hi @Nad-SK , I left several comments here. Can you please take a look at them?

File: files/lang/vi.po [hunk 1]
```diff
@@ -553,7 +580,7 @@ msgid "You have captured an enemy artifact!"
 msgstr "Ngươi đã chiếm được một bảo vật của kẻ thù!"
 
 msgid "Necromancy!"
-msgstr ""
+msgstr "Pháp sư!"
```
[the remark this answers is not in the document]
Comment (Nad-SK) [reply files/lang/vi.po:583]: Fixed!

File: files/lang/vi.po [hunk 2]
```diff
@@ -563,7 +590,7 @@ msgstr ""
 "đã chết để trở lại dưới sự phục vụ của ngươi với tư cách là %{monster}."
 
 msgid "%{name} the %{race}"
-msgstr ""
+msgstr "%{name} chủng tộc %{race}"
```
[the remark this answers is not in the document]
Comment (Nad-SK) [reply files/lang/vi.po:593]: Fixed

File: files/lang/vi.po [hunk 3]
```diff
@@ -605,10 +632,10 @@ msgid "Cancel"
 msgstr "Từ bỏ"
 
 msgid "Hero Screen"
-msgstr ""
+msgstr "Màn hình Tướng"
```
[the remark this answers is not in the document]
Comment (Nad-SK) [reply files/lang/vi.po:635]: Hero mean Anh Hùng, Tướng.
Chinese chess mean Cờ Tướng
So I keep this mean for my translate.

File: files/lang/vi.po [hunk 4]
```diff
@@ -646,7 +673,7 @@ msgid "%{name} states:"
 msgstr "%{name} tình trạng:"
 
 msgid "Captain of %{name} states:"
-msgstr ""
+msgstr "Tình trạng của đội trưởng %{name}:"
```
[the remark this answers is not in the document]
Comment (Nad-SK) [reply files/lang/vi.po:676]: I use this translate.

File: files/lang/vi.po (hunk 1 above)
[the remark this answers is not in the document]
Comment (ihhub) [reply files/lang/vi.po:583]: How is it fixed is nothing was changed?

File: files/lang/vi.po (hunk 4 above)
[the remark this answers is not in the document]
Comment (ihhub) [reply files/lang/vi.po:676]: Status and state as a verb are 2 different things.

File: files/lang/vi.po (hunk 1 above)
[the remark this answers is not in the document]
Comment (Nad-SK) [reply files/lang/vi.po]: Done

File: files/lang/vi.po (hunk 2 above)
[the remark this answers is not in the document]
Comment (Nad-SK) [reply files/lang/vi.po]: Done

**Comment by Nad-SK** [comment]
Hi @ihhub, I checked the errors and fixed it.

File: files/lang/vi.po [hunk 5]
```diff
@@ -2636,7 +2668,7 @@ msgid "Damage"
 msgstr "Thiệt hại"
 
 msgid "HP"
-msgstr ""
+msgstr "Máu"
```
Comment (ihhub) [inline files/lang/vi.po:2671]: Google says that this is **Blood** which is not the same as Health Points (HP).

**Review by ihhub — COMMENTED** [review]
Hi @Nad-SK , I left one more comment here. Could you please take a look?

File: files/lang/vi.po (hunk 5 above)
Comment (Nad-SK) [reply files/lang/vi.po:2671]: OK man, Health Points ~ Điểm Sức khỏe 
In VN we call HP ~ Máu, Mana ~ Phép in some games like Diablo, HoMM series

**Comment by Nad-SK** [comment]
Hi @ihhub, I checked your comment and fixed it.

**Comment by Nad-SK** [comment]
> Hi @Nad-SK , my question about HP wasn't a demand for change but just a question. With your new changes I fear that there would be not enough space in the Well dialog: image

Now what??? I think HP~ Máu is OK for the space. Or no need to translate this, because in VN we know what is HP.

**Comment by Nad-SK** [comment]
Hi @ihhub I translate HP ~ Máu for the space in well dialog and for some sentences with HP like: HP drain, HP regeneration...

**Comment by ihhub** [comment]
@Nad-SK , thank you so much for this update!

**Pull request merged**